pai-management/bootstrap/prometheus/prometheus-configmap.yaml.template

```diff
@@ -44,6 +44,18 @@ data:
         regex: '([^:]+)(:\d*)?'
         replacement: '${1}:{{ clusterinfo['prometheusinfo']['node_exporter_port'] }}'
         target_label: __address__
+    - job_name: 'yarn_exporter'
```

Comment:
This is too ugly, we should use pod discovery instead of node discover to do this.

I already submitted #1047 , in this way other exporter can annotate pod with required annotation, and prometheus can discover them directly.

I think you should remove changes to this file and wait my PR merged and configured yarn-exporter that way.

Reply:
Has modified in this way. Will test later